analytical/utils.py

```diff
@@ -49,6 +49,18 @@ def get_user_from_context(context):
     return None
 
 
+def get_user_is_authenticated(user):
+    """Check if the user is authenticated.
+
+    This is a compatibility function needed to support both Django 1.x and 2.x;
+    Django 2.x turns the function into a proper boolean so function calls will
+    fail."""
```

Comment:
I would prefer to have the `"""` in a separate line, both for the start and the end of the docstring.